data/mumtazcs

```diff
@@ -0,0 +1,52 @@
+{
+ "name": "Mohammad Mumtaz",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web Developer, Student",
+  "tags": ["Open Source", "Content Writer", "Blogging", "Full Stack Developer"],
+  "avatar": "https://github.com/Mumtaz12.png",
+
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/IAmMumtaz",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/withmumtazcs/",
+      "icon": "FaInstagram"
+    },
+   {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohammad-mumtaz-3957b4260/",
+      "icon": "FaLinkedIn"
+    },
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://www.youtube.com/@DeepFakevolution",
+      "icon": "FaYouTube"
```

Comment:
this icon is wrong, it should be `FaYoutube`

Reply:
resolved